developerportal/apps/articles/tests/test_articles.py

```diff
@@ -1,10 +1,10 @@
-from wagtail.tests.utils import WagtailPageTests
+from developerportal.apps.common.test_helpers import PatchedWagtailPageTests
```

Comment:
There are quite a few places where this simple replacement of the base test case happens - it's just because `WagtailPageTests` have a `login()` step that expects Django's `ModelBackend` to be available, but we've dropped it entirely. 

As such `PatchedWagtailPageTests` gets around this missing backend issue in order to make the Wagtail test-helpers in `WagtailPageTests` viable again. You'll see the code change later